crates/matrix-sdk/src/widget/README.md

```diff
@@ -0,0 +1,74 @@
+# Widget Driver
+
+This module implements a widget driver. The widget driver allows a client to give widgets (webviews) access to Matrix via
+a specified `postMessage` API.
+It supports sending and reading events and provides some rudimentary client navigation features.
+There are some additional actions:
```

Comment:
I would also mention that this implements this and that MSC and link to the MSC.